tests/test_chapter03.py

```diff
@@ -53,6 +53,8 @@ def test_digram_counter(self):
         self.assertDictEqual(count, test_count)
 
     @unittest.mock.patch('sys.stdout', new_callable=StringIO)
+    @unittest.mock.patch('src.ch03.p1_digram_counter.DICTIONARY_FILE_PATH',
```

Comment:
This only works if the constant is used in the function's scope. The constant *does* get patched in the module's scope, but if the function doesn't use the constant within its scope, the function loads the module's definition on import.

That was a fun debug session, I assure you.